client/app/bundles/course/assessment/containers/AssessmentForm/translations.intl.js

```diff
@@ -91,20 +91,34 @@ const translations = defineMessages({
       not be immediately shown to the student. To publish all gradings for this assessment, click \
       on the 'Publish Grades' button on the top right of the submissions listing for this assessment.",
   },
+  passwordRequired: {
+    id: 'course.assessment.form.passwordRequired',
+    defaultMessage: 'At least one password is required',
+  },
   passwordProtection: {
     id: 'course.assessment.form.passwordProtection',
     defaultMessage: 'Password Protection',
   },
-  passwordProtectionHint: {
-    id: 'course.assessment.form.passwordProtectionHint',
-    defaultMessage: "When password protection is enabled, students are allowed to access their \
+  viewPasswordHint: {
+    id: 'course.assessment.form.viewPasswordHint',
+    defaultMessage: "When assessment password is enabled, students are required to input the password in order to \
```

Comment:
Feel free to give suggestions on these hints, given that we have 2 kinds of passwords now, it's important to let the user to know the difference between them.